dev-util/pycharm-community/pycharm-community-2024.2.3.ebuild

```diff
@@ -0,0 +1,128 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop readme.gentoo-r1 wrapper xdg-utils
+
+DESCRIPTION="Intelligent Python IDE with unique code assistance and analysis"
+HOMEPAGE="https://www.jetbrains.com/pycharm/"
+SRC_URI="
+	x86? ( https://download.jetbrains.com/python/${P}.tar.gz )
+	amd64? ( https://download.jetbrains.com/python/${P}.tar.gz )
+	arm64? ( https://download.jetbrains.com/python/${P}-aarch64.tar.gz )
+"
+
+if [[ ${PN} == *-professional ]]; then
+	S="${WORKDIR}/${PN/%-professional/}-${PV}"
+fi
+
+LICENSE="Apache-2.0 BSD CDDL MIT-with-advertising"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="+bundled-jdk"
+
+BDEPEND="dev-util/patchelf"
+
+RDEPEND="!bundled-jdk? ( >=virtual/jre-1.8 )
+	app-arch/brotli
+	app-arch/zstd
+	app-crypt/p11-kit
+	dev-libs/fribidi
+	dev-libs/glib
+	dev-libs/json-c
+	dev-libs/libbsd
+	dev-libs/libdbusmenu
+	dev-libs/nss
+	dev-libs/wayland
+	dev-python/pip
+	media-fonts/dejavu
+	media-gfx/graphite2
+	media-libs/alsa-lib
+	media-libs/fontconfig
+	media-libs/freetype:2=
+	media-libs/harfbuzz
+	media-libs/libglvnd
+	media-libs/libjpeg-turbo:0=
+	media-libs/libpng:0=
+	net-libs/gnutls
+	net-print/cups
+	sys-apps/dbus
+	sys-libs/libcap
+	sys-libs/zlib
+	x11-libs/libX11
+	x11-libs/libxcb
+	x11-libs/libXext
+	x11-libs/libXi
+	x11-libs/libXrandr
+	x11-libs/libXrender
+	x11-libs/libXtst
+	x11-libs/pango
+"
+
+RESTRICT="mirror"
+
+QA_PREBUILT="opt/${P}/*"
```

Comment:
`${P}` includes a version too, but these are installed to `/opt/${PN}` resultin in hundred lines of `* QA Notice: Files built without respecting CFLAGS/LDFLAGS have been detected` etc.